apps/web/app/routes/get-started.tsx

```diff
@@ -175,24 +175,8 @@ export default function Component() {
   } = useLoaderData()
 
   return (
-    <div className="m-auto flex max-w-2xl flex-col space-y-8 px-4 py-8 text-gray-200 sm:px-8 sm:py-16">
+    <div className="flex flex-col max-w-2xl px-4 py-8 m-auto space-y-8 text-gray-200 sm:px-8 sm:py-16">
       <Heading>Get Started</Heading>
-      <SubHeading>Dependencies</SubHeading>
-      <p>
-        Make sure you have{' '}
-        <ExternalLink href="https://github.com/colinhacks/zod">
-          Zod
-        </ExternalLink>
-        ,{' '}
-        <ExternalLink href="https://react-hook-form.com/">
-          React Hook Form
-        </ExternalLink>
-        , and{' '}
-        <ExternalLink href="https://github.com/SeasonedSoftware/domain-functions">
-          Domain Functions
-        </ExternalLink>{' '}
-        in your project before using Remix Forms.
-      </p>
```

Comment:
I took the liberty to remove this block of code because I feel it is redundant. The npm installation instructions on line 197 is pretty clear about the dependencies. 

```
<Pre>npm install remix-forms domain-functions zod react-hook-form</Pre>
```

Reply:
Agree, we also have the "Appreciation" section in the very same page so I think it was a good call!